src/Jobs.js

```diff
@@ -69,16 +76,23 @@ class Jobs extends HttpsClient {
      * Starts job waiting to be started
      * @param jobid
      * @param runsample
+     * @param historicalData
      * @returns {Promise.<*>}
      */
-    start(jobid, runsample) {
+    start(jobid, runsample, historicalData) {
```

Comment:
Let's remove the `historicalData` parameter here. It's actually not supported on this endpoint, only on the creation endpoint. I've updated [NB-226](https://discoverorg.atlassian.net/browse/NB-226) to reflect this.